### ISSUE TITLE
Cura dies on startup due to PyQt5/sip compatibility issue

### ISSUE THREAD

**Issue by bwarden** [clearlinux/distribution#1650]
```
$ cura
Error in sys.excepthook:
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.8/site-packages/cura/CrashHandler.py", line 18, in <module>
    from PyQt5.QtCore import QT_VERSION_STR, PYQT_VERSION_STR, QUrl
ImportError: /usr/lib/python3.8/site-packages/PyQt5/QtCore.so: undefined symbol: _ZN8qfloat1610shifttableE, version Qt_5

Original exception was:
Traceback (most recent call last):
  File "/usr/bin/cura", line 140, in <module>
    import Arcus #@UnusedImport
ModuleNotFoundError: No module named 'sip'
```

**Comment by thiagomacieira** [comment]
Known issue, fixed upstream in Qt 5.14.1. Release coming this week.

**Comment by thiagomacieira** [comment]
Fixed with qtbase 5.14.1 update.
